commands.txt

```diff
@@ -1,4 +1,20 @@
-if [ `uname` = 'Darwin' ]; then export SD_CURL_CMD_WRAPPER="eval"; else export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')"; fi
+if [ `uname` = 'Darwin' ]; then
+    export SD_CURL_CMD_WRAPPER="eval"
+    export SD_ZIP_CMD_WRAPPER=""
```

Comment:
we should set $SD_ZIP_ARTIFACTS to "no" in this case if we can't detect `zip` binary installed in the `Darwin` case

Reply:
Checked mac slaves, zip is available https://screwdriver.corp.yahoo.com/projects/216678 https://jenkins.mobile.screwdriver.corp.yahoo.com/jenkins/job/216678-v3-component/BUILD_CONTAINER=xcode9,label=IOS/4/console#console-section-0